### PR TITLE
Fixes spinner

### LINKED ISSUE #38: Spinner is running in command output logs
Related to: https://github.com/pnp/action-cli-deploy/issues/36

### PR DESCRIPTION
**Pull request by milanholemans**
Closes #38

---

@pnp/cli-for-microsoft-365-maintainers I've done a rebuild. The new `setup` command appears in my build lib directory, but not in my package dist directory. Any clue why?

### CODE REVIEW

**Comment by waldekmastykarz** [comment]
I'm seeing the same and wonder if our package script is incorrect. It seems like right now, it's repackaging the contents of `dist/index.js` which it produces itself, so in other words, it's not picking up any code changes. You can confirm it, by deleting `dist/index.js` which will cause the package script to fail.
Instead, I wonder if the package script should be configured as `ncc build lib/main.js -o dist --source-map` so that it's generating `dist/index.js` based on the code that's output to `lib` after running `npm run build`.

@garrytrinder could you please confirm?

**Comment by milanholemans** [comment]
That would make sense indeed, I was kinda confused as to why it crashed when I removed the files under dist.

**Comment by garrytrinder** [comment]
@milanholemans sorry for the delay on this.

The files in the `dist` folder are the files used when executing an action, this is what is used in the runners, so removing it will cause the action to fail.

If you use `npm run dev` during development then any changes made to the `ts` files will produce the `js` files in the `lib` directory, those files are automatically packaged into the `dist` directory. You can run the packaging task explicitly using `npm run package`, just to be sure that everything is packaged.

**Comment by milanholemans** [comment]
Hi @garrytrinder 

That was indeed what I was assuming. But that still doesn't explain the problem I'm facing. I made a change in the code base which is visible in the `lib` directory, but **not** in the `dist` directory. I think something might be wrong with the package script.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pnp/action-cli-login",
-  "version": "3.0.0",
+  "version": "3.0.1",
   "description": "GitHub Action to log in to a tenant using CLI for Microsoft 365",
   "license": "MIT",
   "main": "dist/index.js",
```
Comment (garrytrinder) [inline package.json]: This will fix the `npm run package` script.

```suggestion
  "main": "lib/main.js",
```

**Review by garrytrinder — COMMENTED** [review]
@milanholemans the `main` property in `package.json` is incorrect, not sure why it doesn't throw an error though as its technically pointing to a file that doesn't exist.

**Comment by milanholemans** [comment]
Thank you @garrytrinder, I think it's fixed now.

**Review by garrytrinder — APPROVED** [review]

**Comment by garrytrinder** [comment]
Thanks @milanholemans 

I've merged and released v3.0.1 to the marketplace, I've also added a v3 tag which targets the same commit.

**Pull request merged**